pkg/apis/migration/v1alpha1/migplan_types.go

```diff
@@ -304,23 +294,23 @@ func (r *MigPlan) BuildRegistryImageStream(name string) (*imagev1.ImageStream, e
 			Namespace: VeleroNamespace,
 		},
 	}
-	err := r.UpdateRegistryImageStream(imagestream)
+	err := r.UpdateRegistryImageStream(imagestream, registryImage)
 	return imagestream, err
 }
 
 // Update a Registry ImageStream as desired for the specified cluster.
-func (r *MigPlan) UpdateRegistryImageStream(imagestream *imagev1.ImageStream) error {
+func (r *MigPlan) UpdateRegistryImageStream(imagestream *imagev1.ImageStream, registryImage string) error {
 	imagestream.Spec = imagev1.ImageStreamSpec{
 		LookupPolicy: imagev1.ImageLookupPolicy{Local: false},
 		Tags: []imagev1.TagReference{
 			imagev1.TagReference{
 				Name: "2",
```

Comment:
@sseago is it OK for "2" to be hard-coded here? I think this is in reference to `registry:2`.